app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -290,12 +290,20 @@ BrowserWorld::State::UpdateControllers(bool& aRelayoutWidgets) {
       resizingWidget.reset();
     }
 
+    if (controller.pointer) {
+      controller.pointer->SetVisible(hitWidget.get() != nullptr);
+      controller.pointer->SetHitWidget(hitWidget);
+    }
+    if (hitWidget) {
+      vrb::Matrix translate = vrb::Matrix::Translation(vrb::Vector(hitPoint.x(), hitPoint.y(), 0.001f));
+      controller.pointer->SetTransform(hitWidget->GetTransform().PostMultiply(translate));
```

Comment:
This `if (hitWidget) {` needs to be in the `if(controller.pointer) {` block since it calls `pointer->SetTransform()`